core/common/src/main/java/alluxio/heartbeat/HeartbeatThread.java

```diff
@@ -162,11 +184,38 @@ public void run() {
     }
   }
 
+  private synchronized void updateState() {
+    mCounter.incrementAndGet();
+    mStartTickTime = 0L;
+    mStartHeartbeatTime = 0L;
```

Comment:
why reset to 0 every time?

Reply:
0 stand for that it is not start to heartbeat this period.